.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
   - npm install -g codecov
   - npm install
 script:
+  - npm audit # if npm audit fails, run `npm audit fix` and commit the results
```

Comment:
I did `npm audit --help` to try figure out what is audited.

```
npm audit [--json] [--production]
npm audit fix [--force|--package-lock-only|--dry-run|--production|--only=(dev|prod)]
```

Perhaps it make sense to use `--production` in the `npm audit` to not have it fail on devDependencies be considered. 

## Rubber duck...

In [the documentation for npm audit](https://docs.npmjs.com/cli/audit) I learned that it is scrubbed content of package-lock.json that is submitted for review as compared to the package.json for example. So, if we would do `npm audit` we would realize if package-lock.json could use an update, and if we would do `npm audit fix` we would update the package-lock.json file... And by doing `npm audit fix --force` we would also potentially update the `package.json` file as I understand it.

Okay I think I've concluded now that I agree that we should do this in travis, and ensure our git repo is updated with a seemingly secure package-lock.json at all time. I also now hypothesize that `npm audit fix` to only update package-lock.json given the constraints of `package.json`, while `npm audit fix --force` will also update package.json.

## Questions
- Should we use `--production` in this audit to not bug with script failures? I think yes.
- Should we suggest the `npm audit fix` to be run with `--only=prod` to avoid bothering with dev dependencies? I think we should update it all if we update the prod environment.

Reply:
```suggestion
  - npm audit --production # if npm audit fails, run `npm audit fix` and commit the results
```